apps/opencs/view/world/dialoguesubview.cpp

```diff
@@ -581,7 +583,15 @@ CSVWorld::DialogueSubView::DialogueSubView (const CSMWorld::UniversalId& id, CSM
 
     changeCurrentId(id.getId());
 
-    QWidget *mainWidget = new QWidget(this);
+    //QWidget *mainWidget = new QWidget(this);
+    CSVDoc::SizeHintWidget *mainWidget = new CSVDoc::SizeHintWidget;
+
+    const QRect rect = QApplication::desktop()->screenGeometry(this);
+    int frameHeight = 40; // set a reasonable default
+    QWidget *topLevel = QApplication::topLevelAt(pos());
+    if (topLevel)
+        frameHeight = topLevel->frameGeometry().height() - topLevel->height();
+    mainWidget->setSizeHint(QSize(400, rect.height()-frameHeight)); // FIXME: 400
```

Comment:
There is no easy way to pre-calculate the required width, so I'll just remove the size hint from dialogue subview.